apps/test-suite/tests/Constants.js

```diff
@@ -10,7 +10,7 @@ export function test(t) {
       t.it(`can only use ${v} in the managed workflow`, () => {
         if (
           Constants.appOwnership === 'expo' ||
-          Constants.appOwnership === 'standalone' ||
+          Constants.appOwnership === null ||
```

Comment:
This changes the semantics a little bit. originally if `Constants.appOwnership=null` it goes to the else block, now it goes to if block. is that expected?

Reply:
Yep. This constant can only be `expo` or null, so my best guess is that we never tested the "bare" case and now it does, and it removes the "standalone" case.

Reply:
ci tests are failed because of this change. it turns out that on bare-expo theses two properties are undefined. i reverted this part in https://github.com/expo/expo/pull/26338. let me know if you have anything to add and we can follow up in a separate pr.